openslides_backend/action/actions/motion/create.py

```diff
@@ -20,7 +20,6 @@ class MotionCreate(MotionCreateBase):
 
     schema = DefaultSchema(Motion()).get_create_schema(
         optional_properties=[
-            "title",
```

Comment:
noticed while testing that `title`was doubled in the optional and required properties, so removed it here.